indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -519,3 +519,43 @@ th.i-table {
         }
     }
 }
+
+#track-list {
+    list-style-type: none;
+    padding-left: 0;
+    width: 800px;
```

Comment:
You can remove this rule if you extend from the correct base html template (comment down below).